lib/n_.js

```diff
@@ -18,9 +18,11 @@ var repl = require('repl'),
 // create new pristine `lodash` instance
 var _ = require('lodash').runInContext(server.context);
 
+var lodashToInject = process.argv.indexOf('--fp') === -1 ? _ : require('lodash/fp').runInContext();
```

Comment:
Not sure if I have to add `server.context` as parameter of `runInContext` here or not. I don't understand why `runInContext` is used, actually.